src/radical/pilot/configs/resource_tacc.json

```diff
@@ -1,5 +1,42 @@
 
 {
+    "frontera_interactive": {
+        "description"                 : "Petascale computing system at the Texas Advanced Computing Center (TACC)",
+        "notes"                       : "Specify this resource in the pilot description and run RP within a compute node(s)",
+        "schemas"                     : ["local", "ssh"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "ssh://localhost/",
+            "filesystem_endpoint"     : "sftp://localhost/"
+        },
+        "local"                       :
+        {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+        "resource_manager"            : "SLURM",
+        "cores_per_node"              : 56,
+        "gpus_per_node"               : 0,
+        "agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["SRUN"],
+                                         "SRUN" : {
+                                             "pre_exec_cached": []
+                                         }
+                                        },
+        "pre_bootstrap_1"             :["export RP_APP_TUNNEL_ADDR=144.76.72.175:27017",
+                                        "echo  $RP_APP_TUNNEL_ADDR"],
```

Comment:
is this address valid?
but anyway, this could be removed as well, tunneling is not needed here

Reply:
fixed